ci-scripts/copy_to_archive

```diff
@@ -44,6 +58,7 @@ if [ -d ${IMAGE_DIR}/${INTEL_ARCH} ]; then
     mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
     mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
     mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.manifest $ARCHIVE_DIR
```

Comment:
why have this based on variant? Can't it be a general rule to copy/move it? It should be applicable for any CPU architecture, i.e. also for Jetson and other.